spec/requests/mutations/resend_confirmation_spec.rb

```diff
@@ -26,7 +26,7 @@
 
   context 'when params are correct' do
     it 'sends an email to the user with confirmation url and returns a success message' do
-      expect { post_request }.to(change(ActionMailer::Base.deliveries, :count).by(1))
+      expect { post_request }.to change(ActionMailer::Base.deliveries, :count).by 1
```

Comment:
`by(1)` does use parenthesis but not a big deal for this review

Reply:
Would you like me to change this? I can if you want me to.

Reply:
Changed this since I was in that file anyway.